src/main/java/dwbh/api/domain/input/CreateVoteInput.java

```diff
@@ -29,12 +29,15 @@
   private final transient UUID votingId;
   private final transient String comment;
   private final transient Integer score;
+  private final transient Boolean anonymous;
 
-  private CreateVoteInput(UUID userId, UUID votingId, String comment, Integer score) {
+  private CreateVoteInput(
+      UUID userId, UUID votingId, String comment, Integer score, Boolean anonymous) {
```

Comment:
`Boolean` -> `boolean` . I'd leave `Integer` for the moment as I don't know if it's desirable having a 0 as a default value.

Reply:
Changed